programs/clearing_house/src/controller/pnl.rs

```diff
@@ -90,7 +90,7 @@ pub fn settle_pnl(
     }
 
     validate!(
-        perp_market.status == MarketStatus::Initialized,
+        perp_market.status == MarketStatus::Active,
```

Comment:
should you be able to settle pnl if `FillPaused`, `FundingPaused` ?

Reply:
yea perhaps it should be ok to